client/boot/index.js

```diff
@@ -88,7 +88,7 @@ function init() {
 	} );
 }
 
-function setUpContext( layout, reduxStore ) {
+function setUpContext( reduxStore ) {
 	// Pass the layout so that it is available to all page handlers
```

Comment:
(I removed both comment lines, including the second one that read `and add query and hash objects onto context object`, since there are extra comments to that effect further down.)